src/specBuilder/signal/signalSpecBuilder.ts

```diff
@@ -90,7 +114,9 @@ export const addHighlightedItemSignalEvents = (signals: Signal[], markName: stri
 			...[
 				{
 					events: `@${markName}:mouseover`,
-					update: `${new Array(datumOrder).fill('datum.').join('')}${MARK_ID}`,
+					update:
+						animations && animateFromZero && !needsDisable ? `timerValue === 1 ? ${highlightedUpdate} : null`
```

Comment:
If we switch to a truthy variable, this check becomes a bit more intuitive. 
```suggestion
						animations && animateFromZero && isEnabled ? `timerValue === 1 ? ${highlightedUpdate} : null`
```